airbyte-integrations/connectors/source-drift/source_drift/spec.json

```diff
@@ -4,14 +4,83 @@
     "$schema": "http://json-schema.org/draft-07/schema#",
     "title": "Drift Spec",
     "type": "object",
-    "required": ["access_token"],
-    "additionalProperties": false,
+    "required": ["credentials"],
```

Comment:
remove credentials from required because we need to support of spec files

Reply:
remove credentials from required because we need to support old spec files

Reply:
Done.